models/anomalies/re_data_alerts.sql

```diff
@@ -0,0 +1,17 @@
+select
+    'anomaly' as type,
+    {{ re_data.clean_blacklist('table_name', ['"', '`'], '') }} as model,
+    message,
+    last_value_text as value,
+    time_window_end
+from
+    {{ ref('re_data_alerting') }}
+union all
+
+select
+    'schema_change' as type,
+    {{ re_data.clean_blacklist('table_name', ['"', '`'], '') }} as model,
+    {{ generate_schema_change_message('operation', 'column_name', 'prev_column_name', 'prev_data_type', 'data_type', 'detected_time') }} as message,
+    '' as value,
+    detected_time as time_window_end
+from {{ ref('re_data_schema_changes') }}
```

Comment:
This model generates a formatted message for anomalies and schema changes. The notifications feature in the python project highlighted the need for centralizing this logic within dbt. Now same view on the observability UI can be seen directly by querying the DB 😄

https://github.com/re-data/re-data/issues/191